community-nightlies/deb.sh

```diff
@@ -273,6 +273,8 @@ main ()
 
   echo -n "Importing Citus Data gpg key... "
```

Comment:
For the sake of consistency, can we do the following change?

```suggestion
  echo -n "Importing Citus Data Community nightlies gpg key... "
```